packages/service-clients/end-to-end-tests/azure-client/src/test/audience.spec.ts

```diff
@@ -40,8 +43,22 @@ describe("Fluid audience", () => {
 	 * Expected behavior: container should have a single member upon creation.
 	 */
 	it("can find original member", async () => {
-		const { container, services } = await client.createContainer(schema);
-		const containerId = await container.attach();
+		let containerId: string;
+		let container: IFluidContainer;
+		let services: AzureContainerServices;
+		if (isEphemeral) {
+			const containerResponse: AxiosResponse | undefined = await createContainerFromPayload(
+				ephemeralSummaryTrees.findOriginalMember,
+				"test-user-id-1",
+				"test-user-name-1",
+			);
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+			containerId = containerResponse?.data?.id as string;
```

Comment:
If we take a stricter fail approach in createContainerFromPayload, this shouldn't need as much conditionals/casting.  As it is, this might be casting `undefined` to string which will probably lead to a more mysterious error than just failing earlier.

Could also consider a simple `typeof` check here to blow up if we get something weird back.

Reply:
Updated it to check that response?.data?.id exists and throw an error if not, so that we can do response.data.id in other parts of code